src/lib/serviceRoutines/postRegisterContext.cpp

```diff
@@ -58,6 +58,28 @@ std::string postRegisterContext
 {
   RegisterContextResponse  rcr;
   std::string              answer;
+  RegisterContextRequest*  rcrP = &parseDataP->rcr.res;
+
+  //
+  // Check for isPatterns in request
+  //
+  for (unsigned int crIx = 0; crIx < rcrP->contextRegistrationVector.size(); ++crIx)
+  {
+    ContextRegistration* crP = rcrP->contextRegistrationVector[crIx];
+
+    for (unsigned int eIx = 0; eIx < crP->entityIdVector.size(); ++eIx)
+    {
+      if (isTrue(crP->entityIdVector[eIx]->isPattern))
+      {
+        std::string  details = "isPattern set to true for a registration";
```

Comment:
I'd suggest:

"isPatter set to true for registrations is currently not supported"

to make clear that is not an error, but a kind of limitation at CB.